.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```

Comment:
I would increase this to 90 days. In case we want to push back something to the next quarter.